moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryKnackDataTrackerSync.js

```diff
@@ -106,6 +106,20 @@ const ProjectSummaryKnackDataTrackerSync = ({
           (fieldMap[process.env[regExResult[0]]] = regExResult[1].toLowerCase())
       );
 
+    console.log(project);
+
+    const url = process.env.REACT_APP_KNACK_DATA_TRACKER_URL_BASE + project.project_id;
+
+    const url_payload = {
+      "url": url,
+      "label": project.project_description,
+    };
+
+    body[ process.env.REACT_APP_KNACK_DATA_TRACKER_MOPED_URL ] = url;
+    body[ process.env.REACT_APP_KNACK_DATA_TRACKER_MOPED_LINK_LABEL ] = project.project_description;
+    body[ process.env.REACT_APP_KNACK_DATA_TRACKER_MOPED_LINK_LABEL ] = url_payload;
```

Comment:
Are you using Prettier for automatic formatting with Javascript? I don't think its common to see empty space around values inside a bracket notation.

Reply:
I did fail to lint this, and it absolutely caught this errant whitespace. thanks!

Reply:
I wouldn't call it errant b/c the code still functions fine with or without. I have my code editor set up so that Prettier runs each time I save the file.